stable/sonarqube/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 0.3.6
-appVersion: 6.5
+version: 0.4.0
```

Comment:
I don't think this is a new feature, so it's better to bump the patch level instead.

Reply:
It was not clear what the chart versioning strategy is so I though since this is a major SQ version update, it makes sense to update the chart major.

Reply:
I followed the suggestion. Please put ok to test again.

Reply:
We use SemVer, so major.minor.patch. Major increase on backwards-incompatible changes, minor increase on new features, patch increase on bug fixes (or documentation fixes).